src/common/logging.ts

```diff
@@ -1,22 +1,24 @@
 import {createLogger, format, transports} from 'winston';
-import {getEnv} from './config';
-const Sentry = require('winston-raven-sentry');
+const {Loggly} = require('winston-loggly-bulk');
+import {getEnvironmentVariables} from './config';
+import winston = require('winston');
 
 /**
  * Winston Log Object
  * Send logs to Console if message level <= 'error'.
  * Send logs to Sentry if message level is at least 'info'.
  */
+const envVariables = getEnvironmentVariables();
 const logging =
   createLogger({
-    level: getEnv() === 'production' ? 'info' : 'debug',
+    level: envVariables.env === 'production' ? 'error' : 'debug',
     defaultMeta: {service: 'speedupamerica-frontend'},
```

Comment:
Please include hostname. 

When I look at the logout:
```
info: Starting main.ts {"service":"speedupamerica-frontend","timestamp":"2019-07-17T01:50:00.354Z"}
info: Stopping main.ts {"service":"speedupamerica-frontend","timestamp":"2019-07-17T01:50:00.356Z"}
```
Can we get the message in the JSON as well as the level?

Reply:
Added `message `and `level`.